packages/@okta/vuepress-site/docs/guides/sign-into-web-app-redirect/main/python/reqautheverything.md

```diff
@@ -1,9 +1,9 @@
-To handle auth for all routes within a group, you can use some middleware or the `@before_request` decorator. We use the `@before_request` decorator; add the following to your main `app.py` file:
+You can use a middleware function or the `@before_request` decorator to protect every endpoint so only authenticated users can access anything. For example:
```

Comment:
```suggestion
You can use a middleware function or the `@before_request` decorator to protect every endpoint so that only authenticated users have access. For example:
```